resources/views/emails/userCreated.blade.php

```diff
@@ -3,4 +3,4 @@
 {{ $blueprint->user->username }} just joined {{ app()->url() }}.
 
 To view this new user's profile, please click on following link:
-{{ app()->url() }}/u/{{ $blueprint->user->id }}
+{{ app()->url() }}/u/{{ $blueprint->user->username }}
```

Comment:
Why is this changed ? If the user's username is changed, the URL will no longer be valid. And their username is already present in the 3rd line.

Reply:
Oh... You are right )